app/views/dashboard/_hosts_search_widget.html.erb

```diff
@@ -0,0 +1,27 @@
+<h4 class="header">
```

Comment:
![widget](https://user-images.githubusercontent.com/11807069/40974933-e8a03ae0-68d1-11e8-999e-cc93b1fb78dc.png)
IMO there's too much space, is there an option to set the height ? /cc @tbrisker

Reply:
all widgets are put on a grid right now with a default height of 1 unit - which is defined as this height currently. changing it would mean changing the height of all widgets (e.g. making a unit 1/10th of a current unit and multiplying sized by 10). The widget sizes are afaik only really used for different width (iirc there is a 12 grid, and most widgets are 4-6 units wide)